CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.1.0]
+### Added
+- Add a new "userOptInV2" prop ([#222](https://github.com/MetaMask/smart-transactions-controller/pull/222))
+
+### Changed
+- chore(deps): bump @metamask/network-controller from 15.0.0 to 15.1.0 ([#219](https://github.com/MetaMask/smart-transactions-controller/pull/219))
+- chore(deps-dev): bump @metamask/auto-changelog from 3.4.1 to 3.4.2 ([#221](https://github.com/MetaMask/smart-transactions-controller/pull/221))
```

Comment:
`auto-changelog` is a development dependency. We typically leave these out as only developers care about this. We can also clean up the first item slightly:

``` suggestion
- Bump @metamask/network-controller from 15.0.0 to 15.1.0 ([#219](https://github.com/MetaMask/smart-transactions-controller/pull/219))
```